keyboards/keyhive/maypad/rules.mk

```
@@ -32,4 +32,4 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
 
-LAYOUTS = ortho_5x4 numpad_5x4
+LAYOUTS = ortho_5x4 numpad_5x4 numpad_5x4_split_plus
```

This has to be list of valid community layouts
```suggestion
LAYOUTS = ortho_5x4 numpad_5x4
```

---

Ah, got it, I didn't realize that needed to be the case. Thanks for the help!